administrator/components/com_guidedtours/forms/tour.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form addfieldprefix="Joomla\Component\Guidedtours\Administrator\Field">
+
+	<field
+		name="id"
+		type="text"
+		label="JGLOBAL_FIELD_ID_LABEL"
+		class="readonly"
+		readonly="true"
+		default="0"
+	/>
+	
+	<field
+		name="asset_id"
+		type="hidden"
+		filter="unset"
+		label="JFIELD_ASSET_ID_LABEL"
+	/>
+
+	<field
+		name="title"
+		type="text"
+		label="JGLOBAL_TITLE"
+		size="40"
+		required="true"
+	/>
+
+	<field
+		name="description"
+		type="editor"
+		label="JGLOBAL_DESCRIPTION"
+		filter="JComponentHelper::filterText"
+		buttons="true"
+		hide="readmore,pagebreak"
+	/>
+
+	<field
+		name="published"
+		type="list"
+		label="JSTATUS"
+		default="1"
+		class="form-select-color-state"
+		size="1"
+		validate="options"
+		>
+		<option value="1">JPUBLISHED</option>
+		<option value="0">JUNPUBLISHED</option>
+		<option value="2">JARCHIVED</option>
+		<option value="-2">JTRASHED</option>
+	</field>
+
+	<field
+		name="checked_out"
+		type="hidden"
+		filter="unset"
+		default="0"
+	/>
+
+	<field
+		name="checked_out_time"
+		type="hidden"
+		filter="unset"
+	/>
+
+	<field
+		name="access"
+		type="accesslevel"
+		label="JFIELD_ACCESS_LABEL"
+		filter="UINT"
+		validate="options"
+	/>
+
+	<field
+		name="created_user_id"
+		type="user"
+		label="JGLOBAL_FIELD_CREATED_BY_LABEL"
+		validate="UserId"
+	/>
+
+	<field
+		name="created_time"
+		type="calendar"
+		label="JGLOBAL_CREATED_DATE"
+		translateformat="true"
+		showtime="true"
+		size="22"
+		filter="user_utc"
+	/>
+
+	<field
+		name="modified_user_id"
+		type="user"
+		label="JGLOBAL_FIELD_MODIFIED_BY_LABEL"
+		validate="UserId"
+	/>
+
+	<field
+		name="modified_time"
+		type="calendar"
+		label="JGLOBAL_FIELD_MODIFIED_LABEL"
+		class="readonly"
+		translateformat="true"
+		showtime="true"
+		size="22"
+		filter="user_utc"
+	/>
+
+	<field
+		name="language"
+		type="contentlanguage"
+		label="JFIELD_LANGUAGE_LABEL"
+		>
+		<option value="*">JALL</option>
+	</field>	
+	
+	<field
+		name="url"
+		type="url"
+		label="COM_GUIDEDTOURS_URL_LABEL"
+		size="40"
+		description="COM_GUIDEDTOURS_URL_DESC"
+	/>
+	
+	<field
+		name="rules"
+		type="rules"
+		label="JFIELD_RULES_LABEL"
+		translate_label="false"
+		filter="rules"
+		validate="rules"
+		component="com_guidedtours"
+		section="tour"
+	/>
+	
+	<field
+		name="overlay"
+		type="radio"
+		label="COM_GUIDEDTOURS_OVERLAY_LABEL"
+		layout="joomla.form.field.radio.switcher"
+		default="0"
+		description="COM_GUIDEDTOURS_OVERLAY_DESC"
```

Comment:
These should be ordered
name
type
label
description
.... then everything else